packages/reload-middleware/src/ui5/middleware.ts

```diff
@@ -0,0 +1,53 @@
+import {
+    getLivereloadServer,
+    getConnectLivereload,
+    defaultLiveReloadOpts,
+    defaultConnectLivereloadOpts
+} from '../base';
+import type { ReloaderConfig, HttpsOptions } from '../base';
+import type { MiddlewareParameters } from '@ui5/server';
+import type { RequestHandler } from 'express';
+import { ToolsLogger, UI5ToolingTransport } from '@sap-ux/logger';
+import { resolve } from 'path';
+
+module.exports = async ({ options, middlewareUtil }: MiddlewareParameters<ReloaderConfig>): Promise<RequestHandler> => {
+    const logger = new ToolsLogger({
+        transports: [new UI5ToolingTransport({ moduleName: 'reload-middleware' })]
+    });
+    const rootPath = middlewareUtil.getProject().getRootPath();
+    const sourcePath = middlewareUtil.getProject().getSourcePath();
+    let httpsOpts: HttpsOptions = {
+        key: process.env.FIORI_TOOLS_SSL_KEY,
```

Comment:
Could you please mention the option of using these env variables in README.

Reply:
Yes, will do.

Reply:
Done with https://github.com/SAP/open-ux-tools/pull/1704/commits/f7b7191f4b98d7c912d89bc4bff471c4ee66c8f3